extensions/compose/src/compose-wrapper-generator.ts

```diff
@@ -44,7 +44,7 @@ export class ComposeWrapperGenerator {
     }
 
     // render the template
-    return mustache.render(template, { socketPath, binFolder: this.binFolder });
+    return render(template, { socketPath, binFolder: this.binFolder });
```

Comment:
looks like it's failing using that syntax

Reply:
maybe we should downgrade mustache https://github.com/janl/mustache.js/issues/797 👀

Reply:
I haven't found a nice way to make it work. As it works at runtime and it seems there is an error in the typings i just added a command to ignore the error. If you have any suggestion to make it another way let me know

Reply:
i guess so, it complains that there are no mustache types if i leave the code as it was... but then if i add the types they are missing the default export